package.json

```diff
@@ -11,6 +11,7 @@
     "pretest": "npm run lint && npm run build",
     "test": "floss --path test",
     "unit-test": "floss --path test",
+    "unit-test:debug": "floss --path test '--debug'",
```

Comment:
On second thought @mat could you remove this? I'm adding a PR to add some other testing bits.

Reply:
nice! removed 👍